exporters/trace_zipkin/src/main/java/io/opencensus/exporter/trace/zipkin/ZipkinExporter.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2017, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.exporter.trace.zipkin;
+
+import static com.google.common.base.Preconditions.checkState;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.export.SpanExporter;
+import io.opencensus.trace.export.SpanExporter.Handler;
+import javax.annotation.concurrent.GuardedBy;
+import zipkin.reporter.Sender;
+import zipkin.reporter.SpanEncoder;
+import zipkin.reporter.urlconnection.URLConnectionSender;
+import zipkin2.Span;
+
+/**
+ * An OpenCensus span exporter implementation which exports data to Zipkin.
+ *
+ * <p>Example of usage:
+ *
+ * <pre><code>
+ *   public static void main(String[] args) {
+ *     ZipkinExporter.createAndRegister("http://127.0.0.1:9411/api/v2/spans", "myservicename");
+ *     ... // Do work.
+ *   }
+ * </code></pre>
+ */
+public final class ZipkinExporter {
+
+  private static final String REGISTER_NAME = ZipkinExporter.class.getName();
+  private static final Object monitor = new Object();
+
+  @GuardedBy("monitor")
+  private static Handler handler = null;
+
+  private ZipkinExporter() {
+  }
```

Comment:
Sorry my OCD wants this (I think this is how java formater should format this):
private ZipkinExporter() {}

 If you use IDEA there is a chance that you have the plugin disabled, I remember that after each restart or (maybe update) you have to re-enable it.